src/Event/Productions/SimilaritiesClient.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event\Productions;
+
+use GuzzleHttp\Client;
+
+class SimilaritiesClient
+{
+    /**
+     * @var Client
+     */
+    private $client;
+
+    /**
+     * @var string
+     */
+    private $uri;
+
+    /**
+     * @var string
+     */
+    private $key;
+
+    public function __construct(Client $client, string $uri, string $key)
+    {
+        $this->client = $client;
+        $this->uri = $uri;
+        $this->key = $key;
+    }
+
+    public function excludeTemporarily(array $eventPairs): void
+    {
+        $data['pairs'] = [];
+        /** @var EventPair $pair */
```

Comment:
Nit: Would probably be useful to add a docblock above the method for this, and remove the inline comment. That way it's clear for people looking at the signature of the method that the array should contain `EventPair` objects